app/assets/javascripts/modules/calendars/guider-slot-picker.es6

```diff
@@ -44,14 +54,10 @@
       $(window).on('beforeunload', () => {
         return this.saveWarningMessage;
       });
-
-      $(window).on('unload', () => {
```

Comment:
As far as I can see, Chrome/Firefox/IE11 are all happy with just `beforeunload` event and not `unload`